packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -62,6 +88,20 @@ class ScrollBehavior {
     return null;
   }
 
+  /// A function for carrying scrolling momentum depending on the platform.
+  ///
+  /// The function is applied to the existing scroll velocity when another
+  /// scroll drag is applied in the same direction.
+  ///
+  /// Scrolling momentum is not carried on Android and grows exponentially
+  /// on iOS.
+  MomentumCarryFunction getScrollMomentumCarryFunction(BuildContext context) {
```

Comment:
This seems like something that belongs in the physics, not on the configuration.